iphone/Classes/TiUITextArea.m

```diff
@@ -5,7 +5,7 @@
  * Please see the LICENSE included with this distribution for details.
  */
 #ifdef USE_TI_UITEXTAREA
-
+#import "TiUIAttributedStringProxy.h"
```

Comment:
Everything related to other proxies should be wrapped:
```objc
#ifdef USE_TI_UIATTRIBUTEDSTRING
#import "TiUIAttributedStringProxy.h"
#endif
```
See the TiUITextWidget.m implementation for details.